internal/apiclient/errors.go

```diff
@@ -13,11 +14,23 @@ import (
 // too many things, but it's a start.
 var pathRegex = regexp.MustCompile(`(\w+:)?[\.~\w]*[\/\\]+([^\s:'\"\]]+)|([\w+-]\.\w{2,})`)
 
+var ignoredErrors = []string{
+	"Tag policy check failed",
+	"Policy check failed",
+	"Guardrail check failed",
+}
+
 func ReportCLIError(ctx *config.RunContext, cliErr error, replacePath bool) error {
 	if ctx.Config.IsSelfHosted() {
 		return nil
 	}
 
+	for _, pattern := range ignoredErrors {
+		if strings.Contains(cliErr.Error(), pattern) {
+			return nil
+		}
+	}
+
```

Comment:
Maybe I should do this on `handleCLIError` level instead 🤔 The other two calls of `ReportCLIError` happen for unhandled errors and parsing.